frontend/src/component/playground/Playground/playground.utils.ts

```diff
@@ -125,3 +125,20 @@ export const normalizeCustomContextProperties = (
 
     return output;
 };
+
+export const validateTokenFormat = (token: string): void => {
```

Comment:
Lots of interesting things to unit test with a dedicated test w/o the UI.

E.g
```ts
    it('should not throw an error for a valid token format', () => {
        const validToken = 'project:environment.otherstuff';
        expect(() => validateTokenFormat(validToken)).not.toThrow();
    });

    it('should throw an error if the token does not have a project or environment', () => {
        const invalidToken1 = '.otherstuff';  // Missing project and environment
        const invalidToken2 = 'project.otherstuff';  // Missing ':'
        const invalidToken3 = 'project:.otherstuff';  // Missing environment

        expect(() => validateTokenFormat(invalidToken1)).toThrow('Invalid token format');
        expect(() => validateTokenFormat(invalidToken2)).toThrow('Invalid token format');
        expect(() => validateTokenFormat(invalidToken3)).toThrow('Invalid token format');
    });

    it('should throw an error if the environment is an admin token', () => {
        const adminToken = 'project:*.otherstuff';
        expect(() => validateTokenFormat(adminToken)).toThrow('Admin tokens are not supported in the playground');
    });

    it('should not throw an error if the environment is not an admin token', () => {
        const validToken = 'project:dev.otherstuff';
        expect(() => validateTokenFormat(validToken)).not.toThrow();
    });
```